spec/java/util/LinkedHashSet.main.lsl

```diff
@@ -122,15 +123,10 @@ automaton LinkedHashSetAutomaton
     }
 
 
-    proc _generateKey (visitedKeys: map<Object, Object>): Object
+    @KeepVisible proc _generateKey (unseenKeys: map<Object, Object>): Object
```

Comment:
Can we discard and manually inline this subroutine?

Reply:
Fixed;